### PR TITLE
Support for XinaBox CC03 and CS11

### PR DESCRIPTION
**Pull request by KalbeAbbas**
CC03 is a ATSAMD21G18 Cortex-M0+ breakout in xChips format. CS11 is the same, but with a SD Card Interface. 
https://xinabox.cc/products/cc03
https://xinabox.cc/products/cs11

### CODE REVIEW

**Comment by ladyada** [comment]
please change "CC03" and such to a more descriptive name, such as "XinaBox_CC03" etc

File: ports/atmel-samd/boards/CC03/mpconfigboard.mk [hunk 1]
```diff
@@ -0,0 +1,30 @@
+LD_FILE = boards/samd21x18-bootloader.ld
+USB_VID = 0x239A
+USB_PID = 0x8014
```
Comment (ladyada) [inline ports/atmel-samd/boards/CC03/mpconfigboard.mk]: Do not use our VID/PID

File: ports/atmel-samd/boards/CS11/mpconfigboard.mk [hunk 2]
```diff
@@ -0,0 +1,31 @@
+LD_FILE = boards/samd21x18-bootloader.ld
+USB_VID = 0x239A
+USB_PID = 0x8014
```
Comment (ladyada) [inline ports/atmel-samd/boards/CS11/mpconfigboard.mk]: Do not use the same VID/PID

File: ports/atmel-samd/boards/CS11/pins.c [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+//     { MP_ROM_QSTR(MP_QSTR_A3), MP_ROM_PTR(&pin_PA04) },
+//     { MP_ROM_QSTR(MP_QSTR_A4), MP_ROM_PTR(&pin_PA05) },
+//     { MP_ROM_QSTR(MP_QSTR_A5), MP_ROM_PTR(&pin_PB02) },
+//     { MP_ROM_QSTR(MP_QSTR_D0), MP_ROM_PTR(&pin_PA11) },
+    { MP_ROM_QSTR(MP_QSTR_RX), MP_ROM_PTR(&pin_PA11) },
+//     { MP_ROM_QSTR(MP_QSTR_D1), MP_ROM_PTR(&pin_PA10) },
+    { MP_ROM_QSTR(MP_QSTR_TX), MP_ROM_PTR(&pin_PA10) },
+//     { MP_ROM_QSTR(MP_QSTR_D2), MP_ROM_PTR(&pin_PA14) },
+     { MP_ROM_QSTR(MP_QSTR_CS), MP_ROM_PTR(&pin_PA09) },
+//     { MP_ROM_QSTR(MP_QSTR_D4), MP_ROM_PTR(&pin_PA08) },
```
Comment (ladyada) [inline ports/atmel-samd/boards/CS11/pins.c]: delete unused pins, dont comment them out

**Comment by tannewt** [comment]
Note there is a guide about adding a board to CircuitPython here: https://learn.adafruit.com/how-to-add-a-new-board-to-circuitpython

**Comment by KalbeAbbas** [comment]
Could you please recommend some VID/PID pairs? We want to be able to use with mu editor. Thanks

**Comment by ladyada** [comment]
Hi, have you been  and are you currently shipping these two hardware designs with USB_VID = 0x239A? If not, which VID are you using?

**Comment by KalbeAbbas** [comment]
We are closing the PR and will come back after some upgrades. Thanks

**Comment by ladyada** [comment]
OK @KalbeAbbas @gotfredsen 

please reply to the question:
Hi, have you been and are you currently shipping these two hardware designs with USB_VID = 0x239A? If not, which VID are you using?

**Comment by gotfredsen** [comment]
We are not shipping hardware with any VID. The CircuitPython configuration was loaded with the wrong VID/PID pair which is the reason for the PR withdrawal. The VID/PID pair is only going to be available in the CircuitPython .uf2 file. 
We haven’t allocated a VID/PID pair, but might use:
VID: 0x04D8
PID: 0xED5E
Once again, our hardware is not shipped with a VID/PID pair, it’s only allocated in the software and we will only do a PR once a proper pair has been allocated.
I apologize for the mistake.

**Comment by ladyada** [comment]
Thats good to hear, USB VID and PID are something you cannot borrow from another board. So don't copy and paste these values! If you don't have a VID/PID pair, open up an issue in this CircuitPython repository and request one, we will assist in the process of assigning a PID for use with CircuitPython.

**Comment by KalbeAbbas** [comment]
Hi, we have updated with new VID/PID pairs, they are as:

XinaBox CC03 — (0x04D8, 0xEC72)
XinaBox CS11 — (0x04D8, 0xEC75)

Thanks

**Pull request merged**